src/jit/lsra.cpp

```diff
@@ -8270,7 +8270,7 @@ void LinearScan::resolveRegisters()
             {
                 JITDUMP(" internal");
                 GenTreePtr indNode = nullptr;
-                if (treeNode->OperIsIndir())
+                if (treeNode->OperGet() == GT_IND)
```

Comment:
>if (treeNode->OperGet() == GT_IND) [](start = 16, length = 34)

It might be a good idea to audit all OperIsIndir() uses to see if code really meant to check if it is GT_IND/GT_STOREIND/GT_NULLCHECK